sound/soc/sof/pcm.c

```diff
@@ -147,7 +157,7 @@ static int sof_pcm_hw_params(struct snd_pcm_substream *substream,
 	if (posn_offset > sdev->stream_box.size ||
 	    posn_offset % sizeof(struct sof_ipc_stream_posn) != 0) {
 		dev_err(sdev->dev, "error: got wrong posn offset 0x%x for PCM %d\n",
-			posn_offset, ret);
+			posn_offset, spcm->pcm.pcm_id);
```

Comment:
Should we override the value of 'ret' below when this error happens?

Reply:
@plbossart of course yes. you are hawk-eyed!

Reply:
@plbossart fixed now